api/internal.json

```diff
@@ -134,6 +94,14 @@
       "enter_date": "2019-01-25T00:00:00.000",
       "exit_date": null,
       "rough_exit_date": "Q4 2020"
-    }
+    },
+     {
+      "name": "War of the Spark",
+      "block": null,
+      "code": "WAR",
+      "enter_date": "2019-04-01T00:00:00.000",
+      "exit_date": null,
+      "rough_exit_date": "Q4 2021"
```

Comment:
Is this exit date correct? I would think this should be Q4 2020.

Reply:
That was a typo. Fixed 😄